test/foundry/unit/concrete/accountexecution/TestAccountExecution_ExecuteFromExecutor.t.sol

```diff
@@ -20,8 +20,13 @@ contract TestAccountExecution_ExecuteFromExecutor is Test, SmartAccountTestLab {
         // Install MockExecutor as executor module on BOB_ACCOUNT
         bytes memory callDataInstall =
             abi.encodeWithSelector(IModuleManager.installModule.selector, uint256(2), address(mockExecutor), "");
-        PackedUserOperation[] memory userOpsInstall = prepareExecutionUserOp(
-            BOB, BOB_ACCOUNT, EXECTYPE_DEFAULT, address(BOB_ACCOUNT), 0, callDataInstall
+        
+        
+        Execution[] memory execution = new Execution[](1);
```

Comment:
now that we have Execution array
type of execution target, value and data can be prepared by separate helpers.
erc20 , native transfer, mock protocol, counter etc for example 
more helpers can be created in same fashion to install a module, to make install data for particular module
and we just use those utils to make execution array element

Reply:
instead of doing something like this 

execution[0] = Execution(address(mockExecutor), 0, execCallData);

enconding data for execCallData can have separate sub utils.

or entire util function for preparing specific nature of **Execution**